hathor/consensus/block_consensus.py

```diff
@@ -211,14 +211,8 @@ def update_voided_info(self, block: Block) -> None:
                         storage.indexes.height.update_new_chain(height, block)
                         storage.update_best_block_tips_cache([block.hash])
                         # It is only a re-org if common_block not in heads
-                        if common_block not in heads:
+                        if common_block != head:
```

Comment:
What this case still be possible?

Reply:
Oh, true, so this becomes `assert common_block == head`, or rather I think we don't even need to do the `common_block = self._find_first_parent_in_best_chain(block)` part.